package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "surveyor",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
@rafayet-monon is this mean private repository? 🤔

Reply:
From here - https://docs.npmjs.com/files/package.json#private

It is a way to prevent accidental publication of private repositories to any registry. i.e. `npm registry` or any internal registry. I guess it's not associated with Git repositories.